src/StreamJsonRpc.Tests/JsonRpcTests.cs

```diff
@@ -563,6 +590,21 @@ public static string ServerMethod(string argument)
             return argument + "!";
         }
 
+        public static string TestParameter(JToken token)
+        {
+            return "object";
+        }
+
+        public static string TestParameter(string argument)
+        {
+            return "array";
```

Comment:
I think returning the concatenation of this constant + argument would make for a better assertion in the test that communication succeeded.